agent/consul/server.go

```diff
@@ -1055,26 +1055,23 @@ func (s *Server) setupRaft() error {
 			stable = wal
 			return nil
 		}
-		// Only use WAL if there is no existing raft.db, even if it's enabled.
+
+		// Default to WAL if there is no existing raft.db, even if it's enabled. Log a warning otherwise
```

Comment:
```suggestion
		// Default to WAL. Only use WAL if there is no existing raft.db, even if it's enabled. Log a warning otherwise
```
~ "even if it's enabled part" doesn't make sense following "Default to WAL". IMO both clarifications are valuable, so we could just combine them.